assets/scss/sections/_sweets-foods.scss

```diff
@@ -0,0 +1,45 @@
+.sweets-foods-wrapper{
+    padding:100px 0;
+    box-sizing: border-box;
+    height:120vh;
+    display:flex;
+    .sweets-foods-boxes-wrapper{
+        display: flex;
+        flex-wrap: wrap;
+        height:100%;
+        width:100%;
+        .sweets-foods-box{
+            height:50%;
+            width:33.33%;
+            display:flex;
+            flex-flow: column;
+            justify-content: center;
+            align-items: center;
+            h3, h4, p{
+                margin-bottom: 20px;
+            }
+            h4{
+                text-transform: uppercase;
+                letter-spacing: 4px;
+            }
+            h3{
+                text-align: center;
+            }
+            h5{
+                border-bottom:2px solid #000;
+                text-transform: uppercase;
+                letter-spacing: 2px;
+            }
+            span{
+                display:block;
+                text-align: center;
+                margin-bottom: 5px;
+            }
+            img{
+                height:100%;
+                width:100%;
```

Comment:
Jeśli zdjęcie nie jest kwadratem to nie można ustawiać height: 100% i width: 100%
Z tego co widzę to te zdjęcia to prostokąty. Do zmiany. 
Jedna wartość powinna wyznaczać wysokość zdjęcia, druga powinna wprost proporcjonalnie zwiększać się względem pierwszej

Reply:
Poprawione